python/lsst/ts/scriptqueue/base_script.py

```diff
@@ -19,6 +20,8 @@
 class ScriptState(enum.IntEnum):
     """ScriptState constants.
     """
+    UNKNOWN = 0
+    """Script state is unknown."""
```

Comment:
What is the need for this? I don't think it is possible to output an event with 0 for the script state (though after looking at some crazy output from ATPtg I am not positive) and I see any use in code.

Unless you have a strong need for this please delete it.

Reply:
So, the default value for any topic attribute is 0. 

What happens sometimes is that I get information about the script being on the queue but the script itself is still loading (so I don't have a State). To deal with that, I create an empty topic to represent that, only filling in the `salindex`. Having this here makes it transparent to deal with that situation here:

https://github.com/lsst-ts/ts_scriptqueue/blob/76ee836f3ea72d7f119373bd342de8682dc72605/python/lsst/ts/scriptqueue/ui/request_model.py#L178-L217

Reply:
OK